lib/sycamore/sycamore/transforms/extract_schema.py

```diff
@@ -42,7 +42,7 @@ def extract_properties(self, document: Document) -> Document:
         pass
 
 
-class OpenAISchemaExtractor(SchemaExtractor):
+class LLMSchemaExtractor(SchemaExtractor):
```

Comment:
I would recommend updating the unit tests to use the new LLMSchemaExtractor and LLMPropertyExtractor classes instead of the OpenAI-specific ones.